saleor/dashboard/customer/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+
+from ...userprofile.models import User
+
+
+class CustomerForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        self.user = kwargs.pop('user', None)
+        super().__init__(*args, **kwargs)
+        if self.user == self.instance:
+            self.fields['is_active'].disabled = True
```

Comment:
We could add a comment here explaining why this field is disabled here.

Reply:
Ok